deploy/olm-catalog/smart-gateway-operator/2.0.0/smart-gateway-operator.v2.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,294 @@
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: smart-gateway-operator
+                - name: ANSIBLE_GATHERING
+                  value: explicit
+                - name: SMARTGATEWAY_IMAGE
+                  value: quay.io/infrawatch/smart-gateway:v2.0.0-beta1
+                - name: CORE_SMARTGATEWAY_IMAGE
+                  value: quay.io/infrawatch/sg-core:v3.0.0
+                - name: BRIDGE_SMARTGATEWAY_IMAGE
+                  value: quay.io/infrawatch/sg-bridge:v1.0.0
+                image: quay.io/infrawatch/smart-gateway-operator:v2.0.0
+                imagePullPolicy: Always
+                name: operator
+                resources: {}
+                volumeMounts:
+                - mountPath: /tmp/ansible-operator/runner
+                  name: runner
+              serviceAccountName: smart-gateway-operator
+              volumes:
+              - emptyDir: {}
+                name: runner
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+          - list
+          - patch
+          - watch
+        - apiGroups:
+          - apps
+          resourceNames:
+          - smart-gateway-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - smartgateway.infra.watch
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: smart-gateway-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - servicetelemetry
+  - monitoring
+  - telemetry
+  - notifications
+  - smartgateway
+  labels:
+    name: smart-gateway-operator
+  links:
+  - name: Source Code
+    url: https://github.com/infrawatch/smart-gateway-operator
+  maintainers:
+  - email: support@redhat.com
+    name: Red Hat CloudOps DFG
+  maturity: alpha
+  provider:
+    name: Red Hat
+  replaces: smart-gateway-operator.v1.0.1
```

Comment:
Hey sorry, didn't notice this first pass. You need to bump this to v1.0.2 since it exists now (it was created as part of the API fix for OCP 4.4).

Reply:
There is so much paperwork around the OLM stuff; it's like we need some additional templating an automation. Or at least a checklist of some sort we can reuse